config/locales/sv.yml

```diff
@@ -1252,7 +1252,14 @@ sv:
         message_text:
           expire_alert_html: "Ditt medlemsskap gick ut <span class='date-warning'> %{expire_date}.</span>"
           renew_membership: Du kan förnya ditt medlemsskap och fortsätta dra nytta av medlemsfördelarna genom att logga in och betala.
-          pay_on_acct_page: "Du betalar via "
+          pay_on_acct_page: &pay_on_acct_page "Du betalar via "
+
+      first_membership_fee_owed:
+        subject: Please pay your SHF Membership fee
+        greeting: *email_greeting
+        message_text:
+          fee_is_unpaid: "Please pay your SHF membership fee...."
```

Comment:
renew_membership: Du kan betala ditt medlemskap och börja dra nytta av medlemsfördelarna genom att logga in och betala.